extensions/ql-vscode/src/variant-analysis/variant-analysis-manager.ts

```diff
@@ -315,10 +316,21 @@ export class VariantAnalysisManager
     // Build up details to pass to the functions that run the variant analysis.
     const qlPackRootPath = await findPackRoot(uri.fsPath);
     const qlPackFilePath = await getQlPackFilePath(qlPackRootPath);
+
+    // Open popup to ask for language if not already hardcoded
+    const language = qlPackFilePath
+      ? await askForLanguage(this.cliServer)
+      : await findLanguage(this.cliServer, uri);
+
+    if (!language) {
+      throw new UserCancellationException("Could not determine query language");
+    }
+
```

Comment:
This seems to change the flow of running the MRVA 🤔 For example, on `main` I can just right-click "Run against variant analysis repos" and immediately run the query, but now it asks for a language first:

https://github.com/github/vscode-codeql/assets/42641846/373e72bf-2304-4a15-b99a-d65c79c264af

It's possible for me to select the wrong language (e.g. `cpp` instead of `java`), which causes errors:

![image](https://github.com/github/vscode-codeql/assets/42641846/6f4e583a-7ce8-4a4b-8ea8-2a6036bf15fe)

Reply:
Thanks for catching this! The TL;DR is that I did a stupid 🤦 should be fixed with https://github.com/github/vscode-codeql/pull/3270/commits/c18f1c23862fe4fcadfb3f9dfb312178d1e7a28c